dashboard/app/helpers/levels_helper.rb

```diff
@@ -220,7 +220,7 @@ def app_options
       end
 
     # Blockly caches level properties, whereas this field depends on the user
-    @app_options['teacherMarkdown'] = @level.properties['teacher_markdown'] if current_user.try(:authorized_teacher?)
+    @app_options['teacherMarkdown'] = @level.properties['teacher_markdown'] if current_user.try(:authorized_teacher?) && I18n.en?
```

Comment:
huh. I didn't know this method was a thing! I've been doing `I18n.locale == I18n.default_locale` for this.